webpack/components/FormWidget.js

```diff
@@ -0,0 +1,49 @@
+import React, { Component, PropTypes } from 'react';
+
+export default class FormWidget extends Component {
+  render() {
+    return (
+  <div className="form" id={"form_id_"+this.props.form.id}>
+    <div className="form-container">
+      <ul className="list-inline">
+        <li className="form-title">{this.props.form.name}</li>
+      </ul>
+      <ul className="form-list-group list-inline">
+        <li className="form-list-group-item"><h2 className="form-item-title">Systems</h2><i className="fa fa-info-circle fa-lg form-item-icon" aria-hidden="true"></i><p className="form-item-value">145</p></li>
+        <li className="form-list-group-item"><h2 className="form-item-title">Programs</h2><i className="fa fa-info-circle fa-lg form-item-icon" aria-hidden="true"></i><p className="form-item-value">89</p></li>
+      </ul>
+      <div className="form-info">{"Author: "+this.props.form.createdByEmail}</div>
+    </div>
+    <div className="question-set-details">
+      <ul className="list-inline question-set-items">
+        <li className="question-number"><i className="fa fa-question-circle fa-2x" aria-hidden="true"></i><p>15</p></li>
```

Comment:
This should be able to be populated from something like `{ Form.find(this.form.id).questions.count }`